server/status.go

```diff
@@ -148,15 +152,41 @@ func (s *statusServer) handleLocalStacks(w http.ResponseWriter, r *http.Request)
 // handleNodeStatus handles GET requests for node status.
 func (s *statusServer) handleNodeStatus(w http.ResponseWriter, r *http.Request) {
 	// TODO(shawn) parse node-id in path
-	nodes := &status.NodeList{}
-	b, contentType, err := util.MarshalResponse(r, nodes, []util.EncodingType{util.JSONEncoding})
+
+	startKey := engine.KeyStatusNodePrefix
```

Comment:
I've played with this a little.  HandleFunc() doesn't allow wildcards or ids right now, but I've added handleNodesStatus, which if no id is found, will default to listing them all.  
And I've also added a handler for /nodes (note the lack of '/' at the end), which will always list them all.
It might be worthwhile to add in an external router (there are a few available) that is more RESTful friendly.